docs/src/modules/components/WhatsNewLayout.js

```diff
@@ -17,6 +17,110 @@ import TimelineDot from '@mui/lab/TimelineDot';
 import TimelineOppositeContent from '@mui/lab/TimelineOppositeContent';
 
 const entries = [
+  {
+    title: 'MUI X v8',
+    description: 'Cutting-edge features, enhanced customization, and full support for native ESM.',
+    date: new Date(2025, 3, 17),
+    url: 'https://mui.com/blog/mui-x-v8/',
+    highlightList: [
+      {
+        title: 'Data Grid – Pivoting [Premium]',
+        url: 'https://mui.com/blog/mui-x-v8/#pivoting',
+      },
+      {
+        title: 'Data Grid – Ask Your Table (AI Assistance) [Premium]',
+        url: 'https://mui.com/blog/mui-x-v8/#ask-your-table-ai-assistance-in-the-data-grid',
+      },
+      {
+        title: 'Data Grid – List View [Pro]',
+        url: 'https://mui.com/blog/mui-x-v8/#list-view',
+      },
+      {
+        title: 'Data Grid – Server-Side Aggregation & Lazy Loading [Premium/Pro]',
+        url: 'https://mui.com/blog/mui-x-v8/#server-side-aggregation-and-lazy-loading',
+      },
+      {
+        title: 'Data Grid – Data Source with Support for Editing',
+        url: 'https://mui.com/blog/mui-x-v8/#data-source-with-support-for-editing',
+      },
+      {
+        title: 'Data Grid – Data Source on the Community Plan',
+        url: 'https://mui.com/blog/mui-x-v8/#data-source-on-the-community-plan',
+      },
+      {
+        title: 'Data Grid – Row Spanning',
+        url: 'https://mui.com/blog/mui-x-v8/#row-spanning',
```

Comment:
We can skip this one, as well as the List View, since these features were added in v7